### PR TITLE
ARROW-17681: [CI][Packaging] Update brew dependency glib-utils with glib

### PR DESCRIPTION
**Pull request by raulcd**
It seems this was already updated upstream:
https://github.com/Homebrew/homebrew-core/commit/24861e6936715878878b474f0aaf87c85ea8c23d

### CODE REVIEW

**Comment by raulcd** [comment]
@github-actions crossbow submit homebrew-cpp

**Comment by raulcd** [comment]
@github-actions crossbow submit homebrew-cpp

File: c_glib/Brewfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 brew "autoconf-archive"
-brew "glib-utils"
+brew "glib"
```
Comment (kou) [inline c_glib/Brewfile:19]: Can we remove this line too? I think that `glib` is installed automatically as a dependency of `gobject-introspection`.

**Review by kou — APPROVED** [review]
+1

Thanks!

File: c_glib/Brewfile (hunk 1 above)
Comment (raulcd) [reply c_glib/Brewfile]: Done! Thanks @kou !

**Pull request merged**